### DISCUSSION TITLE
Codes or examples for DeepM&Mnet

### DISCUSSION THREAD

**Discussion by HongtaoBi** [lululxvi/deepxde#1221] [Q&A]
Hello Sir,

I recently learned a lot about PINN and DeepONet and tried to use it for my research. I've read the 2 aritcles about DeepM&Mnet mentioned in the README and I'm very interested in DeepM&Mnet. But as a new beginner, it is quite difficult to code a DeepM&Mnet from the very beginning. I would appreciate it if you can provide some codes or examples about DeepM&Mnet.

Thank you very much!

**Comment by lululxvi** [comment]
You can contact the corresponding author.

**Reply by HongtaoBi** [reply]
Thank you for your response!

**Reply by Jiachen1993** [reply]
> Thank you for your response!

Hi Hongtao, did you get the code. I am also currently learning DeepM&Mnet.
